README.md

```diff
@@ -14,6 +14,12 @@ DataJoint for Python is a high-level programming interface for relational databa
 
 DataJoint was initially developed in 2009 by Dimitri Yatsenko in Andreas Tolias' Lab for the distributed processing and management of large volumes of data streaming from regular experiments. Starting in 2011, DataJoint has been available as an open-source project adopted by other labs and improved through contributions from several developers.
 
+## Release notes
```

Comment:
I like the idea of a change log, but let's put that stuff into a file called `CHANGELOG.md`. Other github projects use that as well.